crates/y-sweet-worker/src/config.rs

```diff
@@ -53,26 +53,28 @@ fn parse_s3_config(env: &Env) -> anyhow::Result<S3Config> {
     Ok(S3Config {
         key: env
             .var(S3_ACCESS_KEY_ID)
-            .map_err(|_| anyhow::anyhow!("ok"))?
+            .map_err(|_| anyhow::anyhow!(format!("${S3_ACCESS_KEY_ID} env var not supplied")))?
```

Comment:
The `anyhow!` macro also does formatting! Although for embedding a const, I would just hard-code the string value. Technically I know this makes it harder to change, but I don't expect to change env var names often, and the real value of using consts for env vars is being able to use a language server to list all the locations where they are used.